logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/incremental/IncrementalSyncManager.kt

```diff
@@ -64,11 +64,14 @@ internal class IncrementalSyncManager(
     private val coroutineExceptionHandler = SyncExceptionHandler({
         incrementalSyncRepository.updateIncrementalSyncState(IncrementalSyncStatus.Pending)
```

Comment:
Maybe we can add one here too for cancellations.

```suggestion
        kaliumLogger.i("Cancellation exception handled in SyncExceptionHandler for IncrementalSyncManager")
        incrementalSyncRepository.updateIncrementalSyncState(IncrementalSyncStatus.Pending)
```